presto-parser/src/main/antlr4/io/prestosql/sql/parser/SqlBase.g4

```diff
@@ -140,6 +140,10 @@ statement
     | DESCRIBE INPUT identifier                                        #describeInput
     | DESCRIBE OUTPUT identifier                                       #describeOutput
     | SET PATH pathSpecification                                       #setPath
+    | UPDATE qualifiedName
+        SET targets+=identifier '=' values+=updateSource
+            (',' targets+=identifier '=' values+=updateSource)*
+        (WHERE where=booleanExpression)?                               #update
```

Comment:
See my comment in the `Update` class about modeling these as "assignments". Take a look at how we do it for `io.prestosql.sql.tree.Property`

Reply:
Changed.